src/components/v5/common/TableWithHeaderAndMeatballMenu/TableWithHeaderAndMeatballMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from 'clsx';
+import React, { PropsWithChildren } from 'react';
+import TableWithMeatballMenu from '../TableWithMeatballMenu';
+import { TableWithHeaderAndMeatballMenuProps } from './types';
+
+const displayName = 'v5.common.TableWithHeaderAndMeatballMenu';
+
+const TableWithHeaderAndMeatballMenu = <T,>({
+  title,
+  headerClassName,
+  additionalHeaderContent,
+  children,
+  emptyContent,
+  ...rest
+}: PropsWithChildren<TableWithHeaderAndMeatballMenuProps<T>>) => (
+  <>
+    <div className={clsx(headerClassName, 'pb-4')}>
+      <div className="flex sm:justify-between sm:items-center sm:flex-row flex-col">
+        <div className="flex items-center">
+          <h4 className="heading-5 mr-3">{title}</h4>
+          {additionalHeaderContent}
+        </div>
+        <div className="flex items-center mt-2.5 sm:mt-0">{children}</div>
```

Comment:
Please add a condition to check if children prop is passed and then render this div or not. If you will leave it like this and won't pass children, then it will add a redundant spacing